openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -347,10 +336,13 @@ def check_account_exists(username=None, email=None):
     """
     conflicts = []
 
-    if email is not None and User.objects.filter(email=email).exists():
+    try:
```

Comment:
I don't see the need to incur the overhead of an exception handler on this call that already works fine.